packages/@glimmer/core/src/environment/delegates.ts

```diff
@@ -58,7 +58,7 @@ export class ClientEnvDelegate extends BaseEnvDelegate {
     this.uselessAnchor = self.document.createElement('a');
   }
 
-  protocolForURL(url: string): string {
+  protocolForURL = (url: string): string => {
```

Comment:
p3/5

Couple of issues:

* In general, I'd prefer to keep the class fields together (makes scanning the file much easier)
* I think we should also make `uselessAnchor` a class field
* Once we address both of those things, we can delete the constructor 😸